elements/lisk-p2p/src/peer/inbound.ts

```diff
@@ -40,7 +44,7 @@ export class InboundPeer extends Peer {
 	protected readonly _handleInboundSocketError: (error: Error) => void;
 	protected readonly _handleInboundSocketClose: (
 		code: number,
-		reason: string,
+		reason: string | undefined,
```

Comment:
Maybe instead of `undefined` we can give a message like `Unknown Reason`?

Reply:
This comes from SC so we can't control this input but good point. I made the change below.